content/en/docs/reference/setup-tools/kubeadm/kubeadm-init.md

```diff
@@ -49,7 +49,7 @@ following steps:
    run there.
 
 1. Generates the token that additional nodes can use to register
-   themselves with the master in the future.  Optionally, the user can provide a
+   themselves with the primary control-plane in the future.  Optionally, the user can provide a
```

Comment:
Is primary even needed here? They should be able to register against any existing control plane instance using the credentials.